### PR TITLE
Second round of review

### PR DESCRIPTION
**Pull request by rebplu**

### CODE REVIEW

**Review by hanscools — APPROVED** [review]

**Review by hanscools — APPROVED** [review]

File: Nie-ontologies/Generic-ontologies/catholic-order.ttl [hunk 1]
```diff
@@ -43,16 +43,16 @@ cathord:Order
 cathord:Friar
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/catholic-order#>;
 	a rdfs:Class;
-	rdfs:label "friar"@en, "Ordensbruder"@de;
+	rdfs:label "friar role"@en, "Ordensbruder"@de;
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/catholic-order.ttl]: TBD

**Review by hanscools — APPROVED** [review]

File: Nie-ontologies/Generic-ontologies/creating.ttl [hunk 2]
```diff
@@ -52,14 +53,14 @@ creating:PersonCreator
...
-	rdfs:label "author role of person"@en, "Autorrolle einer Person"@de; #, "auteur"@fr, "autore"@it;
+	rdfs:label "author role of person"@en, "Autorenrolle einer Person"@de; #, "auteur"@fr, "autore"@it;
 	rdfs:comment """Role of a person being responsible for a creation."""@en;
 	rdfs:subClassOf human:PersonRole.
 
 creating:Creating 
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/creating#>;
 	a rdfs:Class;
-	rdfs:label "creating"@en, "Erstellen"@de; 
+	rdfs:label "creating"@en, "erstellen"@de;
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/creating.ttl]: TBD

File: Nie-ontologies/Generic-ontologies/creating.ttl [hunk 3]
```diff
@@ -82,57 +83,57 @@ creating:CreationGroup
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/creating#>;
 	a rdfs:Class;
 	rdfs:label "creation group"@en, "Gruppe von Erstellungen"@de; 
-	rdfs:comment """Group of creations."""@en;
+	rdfs:comment """Group of person's creation."""@en;
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/creating.ttl]: TBD

File: Nie-ontologies/Generic-ontologies/creating.ttl [hunk 4]
```diff
@@ -82,57 +83,57 @@ creating:CreationGroup
...
 	rdfs:subClassOf rdfs:Resource.
 
 creating:CreationSet
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/creating#>;
 	a rdfs:Class;
 	rdfs:label "creation set"@en, "Reihe von Erstellungen"@de;
-	rdfs:comment """Group of persons who create something belonging together"""@en;
+	rdfs:comment """Set of person's creations."""@en;
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/creating.ttl]: TBD

File: Nie-ontologies/Generic-ontologies/device.ttl [hunk 5]
```diff
@@ -79,11 +79,11 @@ device:ElectronicComputer
 	rdfs:comment """Computer also being an electronic machine."""@en;
 	rdfs:subClassOf device:Computer, device:ElectronicMachine.
 
-device:DigitalComputer #°
+device:DigitalComputer
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/device#>;
 	a rdfs:Class;
 	rdfs:label "digital computer"@en, "digitaler Rechner"@de;
-	rdfs:comment """Computer ...."""@en; #°
+	rdfs:comment """Computer using digital software and data to perform a task."""@en;
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/device.ttl]: Computer using programmed digital instructions

File: Nie-ontologies/Generic-ontologies/document.ttl [hunk 6]
```diff
@@ -259,7 +259,7 @@ document:CitationBibliographicReference
 	rdfs:label "citation bibliographic reference"@en, "bibliographische Referenz eines Zitats"@de; # , "référence bibliographique"@fr, "riferimenti bibliografici"@it;
 	rdfs:comment """All information, structured along certain composition rules, to identify a text expression part as citation of a part of another text expression, the latter being published or not. """@en;
 	skos:note """There is not 1 canonical way to write a citation bibliographic reference"""@en;
-	rdfs:subClassOf text-structure:ContentStructure.
+	rdfs:subClassOf document:BibliographicReference.
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/document.ttl]: TBD

**Review by hanscools — APPROVED** [review]

**Review by hanscools — APPROVED** [review]

**Review by hanscools — APPROVED** [review]

**Review by hanscools — APPROVED** [review]

**Review by hanscools — APPROVED** [review]

**Review by hanscools — APPROVED** [review]

**Review by hanscools — APPROVED** [review]

File: Nie-ontologies/Generic-ontologies/event.ttl [hunk 7]
```diff
@@ -97,7 +98,7 @@ event:Staging
 	a rdfs:Class;
 	rdfs:label "staging"@en, "Etappierung"@de;
 	rdfs:comment """Determining the extent of a process."""@en;
-	rdfs:subClassOf event:Action.
+	rdfs:subClassOf event:Process.
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/event.ttl]: TBD

File: Nie-ontologies/Generic-ontologies/human-geography.ttl [hunk 8]
```diff
@@ -46,9 +47,37 @@ humgeo:State
...
+	a rdfs:Class;
+	rdfs:label "nation state"@en, "Nationalstaat"@de;
+	rdfs:comment """Political organization of a nation with an own government."""@en;
+	rdfs:subClassOf humgeo:State.
+
+humgeo:SovereignState
+	rdfs:isDefinedBy <http://e-editiones.ch/ontology/human-geography#>;
+	a rdfs:Class;
+	rdfs:label "sovereign state"@en, "Souveränstaat"@de;
+	rdfs:comment """Political organization of a nation with an own government."""@en;
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/human-geography.ttl]: Comment to be adapted.

File: Nie-ontologies/Generic-ontologies/human-geography.ttl [hunk 9]
```diff
@@ -46,9 +47,37 @@ humgeo:State
...
+	rdfs:subClassOf humgeo:State.
+
+
+humgeo:FederalState
+	rdfs:isDefinedBy <http://e-editiones.ch/ontology/human-geography#>;
+	a rdfs:Class;
+	rdfs:label "federal state"@en, "Bundesstaat"@de;
+	rdfs:comment """Political organization of a nation with an own government."""@en;
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/human-geography.ttl]: Comment to be adapted.

File: Nie-ontologies/Generic-ontologies/human-geography.ttl [hunk 10]
```diff
@@ -46,9 +47,37 @@ humgeo:State
...
+	rdfs:subClassOf humgeo:State.
+
+humgeo:Government
+	rdfs:isDefinedBy <http://e-editiones.ch/ontology/human-geography#>;
+	a rdfs:Class;
+	rdfs:label "government"@en, "Regierung"@de;
+	rdfs:comment """Exercises authority over a particular territory."""@en.
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/human-geography.ttl]: TBD

File: Nie-ontologies/Generic-ontologies/human-geography.ttl [hunk 11]
```diff
@@ -59,11 +88,24 @@ humgeo:Monarchy #°
...
 	rdfs:comment """Role of a person being a sovereign head of state in a monarchy."""@en;
 	rdfs:subClassOf human:PersonRole.
 
-# king, queen
+
+humgeo:King
+	rdfs:isDefinedBy <http://e-editiones.ch/ontology/human-geography#>;
+	a rdfs:Class;
+	rdfs:label "king role"@en, "Königrolle"@de, "rôle de roi"@fr;
+	rdfs:comment """Role of a man being the absolute ruler in a monarchy."""@en;
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/human-geography.ttl]: TBD

File: Nie-ontologies/Generic-ontologies/human-geography.ttl [hunk 12]
```diff
@@ -59,11 +88,24 @@ humgeo:Monarchy #°
...
+	rdfs:subClassOf human:Monarch.
+
+humgeo:Queen
+	rdfs:isDefinedBy <http://e-editiones.ch/ontology/human-geography#>;
+	a rdfs:Class;
+	rdfs:label "queen role"@en, "Königinrolle"@de, "rôle de reine"@fr;
+	rdfs:comment """Role of a woman being the absolute ruler in a monarchy."""@en;
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/human-geography.ttl]: TBD

File: Nie-ontologies/Generic-ontologies/human-geography.ttl [hunk 13]
```diff
@@ -72,15 +114,19 @@ humgeo:Empire
...
-# humgeo:NationState: nation-state
-
-# humgeo:SovereignState
-
-# humgeo:FederatedState
+humgeo:Emperor
+	rdfs:isDefinedBy <http://e-editiones.ch/ontology/human-geography#>;
+	a rdfs:Class;
+	rdfs:label "emperor role"@en, "Kaiserrolle"@de, "rôle d'empereur"@fr;
+	rdfs:comment """Role of a man being a sovereign head of an empire."""@en;
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/human-geography.ttl]: TBD

File: Nie-ontologies/Generic-ontologies/human-geography.ttl [hunk 14]
```diff
@@ -72,15 +114,19 @@ humgeo:Empire
...
+	rdfs:subClassOf human:PersonRole.
 
-# humgeo:Government: exercises authority over a particular territory
+humgeo:Empress
+	rdfs:isDefinedBy <http://e-editiones.ch/ontology/human-geography#>;
+	a rdfs:Class;
+	rdfs:label "empress role"@en, "Kaiserinrolle"@de, "rôle d'impératrice"@fr;
+	rdfs:comment """Role of a woman being a sovereign head of an empire."""@en;
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/human-geography.ttl]: TBD

**Review by hanscools — APPROVED** [review]

**Review by hanscools — APPROVED** [review]

**Review by hanscools — APPROVED** [review]

**Review by hanscools — APPROVED** [review]

**Review by hanscools — APPROVED** [review]

**Review by hanscools — APPROVED** [review]

**Review by hanscools — APPROVED** [review]

**Review by hanscools — APPROVED** [review]

File: Nie-ontologies/Generic-ontologies/human-geography.ttl [hunk 15]
```diff
@@ -76,7 +76,8 @@ humgeo:Government
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/human-geography#>;
 	a rdfs:Class;
 	rdfs:label "government"@en, "Regierung"@de;
-	rdfs:comment """Exercises authority over a particular territory."""@en.
+	rdfs:comment """Group of politicians which exercise authority over a particular territory."""@en.
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/human-geography.ttl]: "One or more persons in a politician role exercising authority over a particular territory."
reb: Done

**Review by hanscools — CHANGES_REQUESTED** [review]

**Comment by rebplu** [comment]
I resolved all comments from yesterday, because these commits were all outdated and I solved it already (see commits from last thursday). Here the overview of all changes which at this stage are made with this pull request: 
[https://github.com/nie-ine/Ontologies/pull/13/files#diff-0a06259006bdf511ea7c7e3d663a48afd619381c508f7f1a0c25b9c41cb9203f](https://github.com/nie-ine/Ontologies/pull/13/files#diff-0a06259006bdf511ea7c7e3d663a48afd619381c508f7f1a0c25b9c41cb9203f)

If everything is ok, could you merge untill tomorrow, so that I can move on?

**Review by hanscools — APPROVED** [review]

**Pull request merged**